src/commitizen/adapter.js

```diff
@@ -78,6 +78,7 @@ function getNearestNodeModulesDirectory(options) {
   let nodeModulesDirectories = findNodeModules(options);
   
    // Make sure we find a node_modules folder
+  /* istanbul ignore else */
```

Comment:
I agree that we shouldn't add the tests now, but isn't it better to not ignore them now, so that we will be reminded to do it at some point? and to not falsely advertise 100% test coverage?